doc.go

```diff
@@ -0,0 +1,17 @@
+/*package bluto is redis library wraper over redigo which add type safty, chained command and
+managed connection pool which makes it easier for developer to use the library and reduce runtime bugs.
+
+bluto instance is compilitly thread-safe which means you can pass bluto object
+to diffretent go-routines without race condition it also automaticly return finished
```

Comment:
go-routines => goroutines
without race condition it => without race condition, it
automaticly => automatically
return => returns